packages/patternfly-4/content/design-guidelines/usage-and-behavior/text-input/design.md

```diff
@@ -0,0 +1,42 @@
+---
+path: "/design-guidelines/usage-and-behavior/text-input"
+---
+# Text input
+Use text input to gather free-form text from a user.
+
+## Elements
+
+![Elements](img/input-elements3.png)
+
+1. Label
+2. Input field
+3. Help
```

Comment:
Maybe say, "Help can be provided in the form of help text below the field or a popover (infotip). Link to more details below?